frontend/public/components/cloud-services/clusterserviceversion.tsx

```diff
@@ -49,7 +49,7 @@ export const ClusterServiceVersionRow = withFallback<ClusterServiceVersionRowPro
 
   const installStatus = obj.status && obj.status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded
     ? <span>Enabled</span>
-    : <span className="co-error"><i className="fa fa-times-circle co-icon-space-r" /> {obj.status.reason}</span>;
+    : <span className="co-error"><i className="fa fa-times-circle co-icon-space-r" /> {_.get(obj, 'status.reason', ClusterServiceVersionPhase.CSVPhaseUnknown)}</span>;
```

Comment:
Fixes a NPE when `status` block is `undefined` (before the OLM Operator has time to process it).